Mathlib/Probability/Moments.lean

```diff
@@ -282,6 +287,27 @@ theorem iIndepFun.cgf_sum {X : ι → Ω → ℝ}
   · rw [h_indep.mgf_sum h_meas]
   · exact (mgf_pos (h_int j hj)).ne'
 
+theorem mgf_ident_distrib (X Y : Ω → ℝ) (hident : IdentDistrib X Y μ μ) :
```

Comment:
Instead of having `X` and `Y` defined on the same space, could you have `X` defined on Omega and `Y` defined on Omega'?

Reply:
Thank you, I used your suggestion.